polaris-react/src/components/TextField/TextField.scss

```diff
@@ -75,6 +75,7 @@ $spinner-icon-size: 12px;
 
   > .Backdrop {
     background-color: var(--p-surface-disabled);
+    border-color: var(--p-border-subdued);
```

Comment:
adding this so that the disabled state border can match the original as it doesn't need to have the same high contrast when disabled

Reply:
Thanks for doing this. While the html attribute disabled does remove the element from keyboard navigation, it does not remove it from screen reader navigation so this fix is helpful there to make sure the focus indicator is still visible.